nexuse2e-webapp/src/main/frontend/src/app/pipes/caps-to-title-case.pipe.ts

```diff
@@ -0,0 +1,22 @@
+import { Pipe, PipeTransform } from "@angular/core";
+import { TitleCasePipe } from "@angular/common";
+
+@Pipe({
+  name: "capsToTitleCase",
+  pure: true,
+})
+export class CapsToTitleCasePipe implements PipeTransform {
+  transform(value: unknown): unknown {
+    if (typeof value === "string") {
+      const titleCasePipe = new TitleCasePipe();
+      const uppercase = value.toUpperCase();
+      if (value === uppercase) {
+        return titleCasePipe.transform(value) as string;
+      } else {
+        return value as string;
+      }
+    } else {
+      return value;
```

Comment:
The angular own Pipes are throw if the argument is a not supported, see
https://github.com/angular/angular/blob/12.0.1/packages/common/src/pipes/case_conversion_pipes.ts#L11-L42
Then we can provide a return type. And I think the null handling is also nice done there

Reply:
Our pipe should transform string values, but other value types should be returned as they are.